templates/ui-package/package.json

```diff
@@ -14,14 +14,60 @@
     "url": "https://github.com/LabShare/<%= appNameSlug %>/issues"
   },
   "license": "MIT",
-  "scripts": {},
+  "scripts": {
+    "test": "karma start ./test/karma.conf.js",
+    "lint": "tslint -p tsconfig.json -c tslint.json",
+    "lint:fix": "tslint --fix -p tsconfig.json -c tslint.json",
+    "build": "lsc build web",
+    "commitmsg": "commitlint -e $GIT_PARAMS",
+    "semantic-release": "semantic-release"
+  },
   "packageDependencies": {
     "@labshare/shell": "*"
   },
   "dependencies": {
-    "@labshare/shell": "https://github.com/LabShare/shell",
-    "lodash": "^4.6.1"
+    "@labshare/shell": "^3.0.8",
+    "@labshare/ng-core": "^3.0.4",
+    "lodash": "^4.6.1",
+    "hammerjs": "^2.0.8",
+    "@angular/core": "6.0.4",
+    "@angular/common": "6.0.4"
   },
   "devDependencies": {
+    "@angular/compiler": "6.0.4",
+    "@angular/compiler-cli": "6.0.4",
+    "@angular/platform-browser": "6.0.4",
+    "@angular/platform-browser-dynamic": "6.0.4",
+    "@commitlint/cli": "^6.2.0",
+    "@commitlint/config-conventional": "^6.1.3",
+    "@labshare/semantic-release-config": "^1.0.0",
+    "@storybook/addon-actions": "3.4.6",
```

Comment:
@rafaelcalpena Since storybook has not been incorporated in the workflow, we can omit it for now.